### PR TITLE
Wrap more content in KPageContainer

### LINKED ISSUE #5242: KPageContainer or panel shadows not used consistently (and nowhere in Learn page)
### Observed behavior

The Learn page is flatter than the other coach/admin pages because the main content is not surrounded by the KPageContainer.

### Expected behavior

Learn Pages are surrounded by the KPageContainer

### Context
0.12rc1

### PR DESCRIPTION
**Pull request by jonboiser**
### Summary

1. Wraps some pages in the Kpagecontainer, which is a centered white container with a box shadow.
1. Re-styles exam and exercise reports to match a new design

This results in a significant visual change to

**Resource previewing (in Coach pages) and viewing (in Learn pages)**

Screen Shot 2019-04-23 at 12 12 07 PM

**Quiz reports**
Screen Shot 2019-04-23 at 12 11 41 PM

**Exercise reports**

Screen Shot 2019-04-23 at 12 28 02 PM

**Quiz viewer**

Screen Shot 2019-04-23 at 11 30 46 AM

**Profile page**

Screen Shot 2019-04-23 at 12 30 14 PM

### Reviewer guidance

1. Go through the app and make sure the new style is applied everywhere it's supposed to be, especially in coach reports for quizzes and exercises, and resource (pre)viewing pages.

### References

Fixes #5242 

----

### Contributor Checklist

PR process:

- [x] PR has the correct target branch and milestone
- [x] PR has 'needs review' or 'work-in-progress' label
- [x] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label
- [ ] If this includes an internal dependency change, a link to the diff is provided

Testing:

- [ ] Contributor has fully tested the PR manually
- [ ] If there are any front-end changes, before/after screenshots are included
- [ ] Critical user journeys are covered by Gherkin stories
- [ ] Critical and brittle code paths are covered by unit tests

### Reviewer Checklist

- Automated test coverage is satisfactory
- PR is fully functional
- PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- External dependency files were updated if necessary (`yarn` and `pip`)
- Documentation is updated
- Contributor is in AUTHORS.md

### CODE REVIEW

File: kolibri/core/assets/src/views/ExamReport/PageStatus.vue [hunk 1]
```diff
@@ -45,6 +53,8 @@
   import ElapsedTime from 'kolibri.coreVue.components.ElapsedTime';
   import KLabeledIcon from 'kolibri.coreVue.components.KLabeledIcon';
   import KIcon from 'kolibri.coreVue.components.KIcon';
+  import HeaderTableRow from '../../../../../plugins/coach/assets/src/views/common/HeaderTable/HeaderTableRow';
+  import HeaderTable from '../../../../../plugins/coach/assets/src/views/common/HeaderTable';
```
Comment (indirectlylit) [inline kolibri/core/assets/src/views/ExamReport/PageStatus.vue]: no can do... you'll need to elevate this to a core component

**Review by nucleogenesis — COMMENTED** [review]
I have only found one area of concern.

Kolibri_PR-5411_CoachesPreviewPDF

When in Coaches pages, previewing a resource that is a PDF, the license, author and copyright information is above the preview of the content. In your example image, that is not the case. 

If that is expected, then this shouldn't be a blocking comment.

Everything else looks great.

**Comment by jonboiser** [comment]
That's a good catch. The Coach Lessons page is in LessonContentPreviewPage.vue, and the Learner Page is in ContentPage.vue, but the functionality is supposed to be similar, so maybe there is an opportunity to consolidate these components later.

But the layout for the coach version of this page is the way it was initially designed (metadata on the top, content below).

**Review by nucleogenesis — APPROVED** [review]
Sounds good - everything looks good to me!

**Pull request merged**